lilio/calendar.py

```diff
@@ -70,16 +70,17 @@ def __init__(
 
     @property
     def is_target(self):
+        """Returns whether this Intervals is a target interval."""
```

Comment:
```suggestion
        """Return whether this Intervals is a target interval."""
```

Just to be consistent with the imperative mood requirement from linter.